airlock/permissions.py

```diff
@@ -186,3 +186,64 @@ def user_can_withdraw_file_from_request(
     except exceptions.RequestPermissionDenied:
         return False
     return True
+
+
+def check_user_can_review_file(user: User, request: "ReleaseRequest", relpath: UrlPath):
+    try:
+        check_user_can_review_request(user, request)
+    except exceptions.RequestPermissionDenied as exc:
+        raise exceptions.RequestReviewDenied(str(exc))
+    policies.check_can_review_request(request)
+    policies.check_can_review_file_on_request(request, relpath)
```

Comment:
should we move `policies.check_can_review_request(request)` into the body of `policies.check_can_review_file_on_request()`?

Reply:
oh yes, good shout